TLM/TLM/Custom/PathFinding/CustomPathFind_Current.cs

```diff
@@ -0,0 +1,4345 @@
+﻿namespace TrafficManager.Custom.PathFinding {
```

Comment:
You can rename it to CustomPathFind

Reply:
I had intent to experiment with A* and dual A* later. OK renamed.